### PR TITLE
fixed DeprecationWarning in stripmapWrapper and SentinelWrapper

### PR DESCRIPTION
**Pull request by mirzaees**
- used `import importlib` instead of `import imp` and modified `__import` method based on new module

- changed `config.readfp` to `config.read_file` in line 25

### CODE REVIEW

**Comment by piyushrpt** [comment]
@mirzaees This is great. The same patterns also show up in configuration/buildHelper.py . Would it be possible to update those? Will result in much shorter build logs to read and debug on the automated CI reports.

**Comment by mirzaees** [comment]
@piyushrpt off course, I have not seen those from configuration/buildHelper.py yet but will try to fix them.

**Comment by piyushrpt** [comment]
I have seen the same messages show up when building isce with scons.

**Comment by mirzaees** [comment]
@piyushrpt 
fixed configuration/buildHelper.py.

**Comment by piyushrpt** [comment]
@mirzaees Hmmm .. unfortunately thats not installing the help files and the unit tests to fail (https://circleci.com/gh/isce-framework/isce2/tree/pull%2F36). Don't worry about it. I copied over your suggested changes and will play with it to see what is happening. The help generator is finicky and we should probably restructure that a little bit anyway. 

 If you want to roll back the last commit. We can go ahead and merge it.

File: configuration/buildHelper.py [hunk 1]
```diff
@@ -98,10 +98,10 @@ def main(factoryFile,package,buildDir):
 #    import isce
     import filecmp
     try:
-        import imp
+        import importlib
         factoryFile = os.path.abspath(factoryFile)
-        mod = imp.find_module(factoryFile.replace('.py',''))
-        factModule = imp.load_module(factoryFile.replace('.py',''),mod[0],mod[1],mod[2])
+        mod = importlib.util.find_spec(factoryFile.replace('.py',''))
```
Comment (piyushrpt) [inline configuration/buildHelper.py]:
```suggestion
        mod = importlib.util.spec_from_file_location('.', factoryFile)
```

This should fix the helper errors.

**Comment by mirzaees** [comment]
yes! Thank you @piyushrpt for your suggestion

**Comment by piyushrpt** [comment]
Thanks for the contribution.

**Pull request merged**